assests/javascript/index.js

```diff
@@ -44,6 +51,68 @@ async function addTask(task) {
   }
 }
 
+async function getAllTasks() {
+  try {
+    const response = await fetch(`${baseUrl}/todos`);
+    const {data} = await response.json();
+
+    return sortTasks(data);
```

Comment:
This function should not care about sorting. This is better to be called inside `showAllTasks` instead.